src/card.css

```
@@ -0,0 +1,106 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px var(--default-padding)
+    var(--default-padding)
+    var(--default-padding);
+  width: calc(200px - 2 * var(--default-padding) - 2 * var(--default-border));
+}
+  
+.container__card {
+  border-radius: 5px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+}
+  
+.card__image {
+  height: 134px;
+  margin-bottom: 40px;
+  margin-right: 3px;
+  margin-left: 3px;
+}
+  
+.card__naming {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+  
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+  
+.card__code {
+  margin: 0;
+  margin-bottom: 16px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+  
+.card__count-rew {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+  color: #060b35;
+}
+  
+.card__price {
+  display: flex;
+  margin: 0;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+  
+.card__price-price {
```

you wrote a lot of redundant code here
```suggestion
.card__price-price {color: #616070;
  font-size: 12px;
  line-height: 18px;}
```